.github/workflows/dotnet.yml

```diff
@@ -0,0 +1,84 @@
+# This workflow will build a .NET project
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-net
+
+name: .NET
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+env:
+  caliburn_sln : "src\\caliburn.micro.sln"
+  caliburn_tutorial: "samples\\tutorals\\WPF\\Wpf.Tutorial\\Caliburn.Micro.Tutorial.App.sln"
+  caliburn_setup: "samples\\setup\\setup.sln"
+  caliburn_features: "samples\\features\\features.sln"
+  package_feed: "https://nuget.pkg.github.com/caliburn-micro/index.json"
+  nuget_folder: "\\packages"
+  
+jobs:
+  build:
+
+    runs-on:  windows-2022
+
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0 # avoid shallow clone so nbgv can do its work
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v4
+      with:
+        dotnet-version: 8.0.x
+    - name: Setup Java SDK
+      uses: actions/setup-java@v4
+      with:
+        distribution: 'microsoft' 
+        java-version: 11
```

Comment:
Use `java-version: 17` or even newer? Like in [codeql-analysis.yml](https://github.com/Caliburn-Micro/Caliburn.Micro/pull/891/files#diff-63bd641104d10e25f141d518a16b22a151d125e12701df2f9e79734b23b90188)

Reply:
done